src/sdk/Player.ts

```diff
@@ -402,13 +402,14 @@ export class Player extends Unit {
   }
 
   activatePrayers() {
-    this.lastOverhead = this.overhead;
     this.overhead = this.prayerController.overhead();
-    if (this.lastOverhead && !this.overhead) {
-      this.lastOverhead.playOffSound();
-    } else if (this.lastOverhead !== this.overhead) {
-      this.overhead.playOnSound();
-    }
+    this.prayerController.prayers.forEach(prayer => {
```

Comment:
Multiple prayer sounds can be played at once, if activated/deactivated on the same tick.